GameServerLib/Logic/GameObjects/Other/MovementVector.cs

```diff
@@ -18,13 +18,13 @@ public MovementVector(short x, short y)
 
         public MovementVector(float x, float y)
         {
-            this.x = FormatCoordinate(x, _game.Map.NavGrid.MapHeight / 2);
-            this.y = FormatCoordinate(y, _game.Map.NavGrid.MapWidth / 2);
```

Comment:
Before you ask, I also don't know why these two are inverted, I kept them inverted.